setup.py

```diff
@@ -18,6 +18,7 @@
           'colorama',
           'argparse',
           'configparser',
+          'BeautifulSoup',
```

Comment:
Are you sure that your pip in your virtual environment is pip3? BeautifulSoup is imported from bs4 in Python3.
$ pip3 install beautifulsoup
Collecting beautifulsoup
  Using cached BeautifulSoup-3.2.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-_59qsyh8/beautifulsoup/setup.py", line 22
        print "Unit tests have failed!"
                                      ^
    SyntaxError: Missing parentheses in call to 'print'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-_59qsyh8/beautifulsoup/